### PR TITLE
Add a new status field apiServerURL for managed cluster info

### PR DESCRIPTION
**Pull request by zhujian7**

### CODE REVIEW

File: clusterinfo/v1beta1/types.go [hunk 1]
```diff
@@ -218,6 +218,11 @@ type OCPDistributionInfo struct {
 	// Controller will sync this field to managedcluster's ManagedClusterClientConfigs
 	// +optional
 	ManagedClusterClientConfig ClientConfig `json:"managedClusterClientConfig,omitempty"`
+
+	// apiServerURL is a valid URI with scheme 'https', address and
+	// optionally a port (defaulting to 443).  apiServerURL can be used by components like the web console
+	// to tell users where to find the Kubernetes API.
+	APIServerURL string `json:"apiServerURL"`
```
Comment (qiujian16) [inline clusterinfo/v1beta1/types.go:225]: LastAppliedAPIServerURL

File: clusterinfo/v1beta1/types.go [hunk 2]
```diff
@@ -218,6 +218,11 @@ type OCPDistributionInfo struct {
 	// Controller will sync this field to managedcluster's ManagedClusterClientConfigs
 	// +optional
 	ManagedClusterClientConfig ClientConfig `json:"managedClusterClientConfig,omitempty"`
+
+	// apiServerURL is a valid URI with scheme 'https', address and
+	// optionally a port (defaulting to 443).  apiServerURL can be used by components like the web console
```
Comment (qiujian16) [inline clusterinfo/v1beta1/types.go:223]: this is the apiserverurl that has been applied to the managedcluster resource successfully

**Comment by zhujian7** [comment]
@qiujian16 PTAL

**Pull request merged**